src/main/daml/Daml/Finance/Lifecycle/Event/Distribution.daml

```diff
@@ -3,7 +3,7 @@
 
 module Daml.Finance.Lifecycle.Event.Distribution where
 
-import Daml.Finance.Interface.Lifecycle.Event qualified as Event (I, View(..))
+import Daml.Finance.Interface.Lifecycle.Event qualified as BaseEvent (I, View(..))
```

Comment:
This change seems unrelated to the existing PR, maybe worth separating it out?

Reply:
will revert, did not indent to change that in this PR